velox/type/Filter.cpp

```diff
@@ -1054,6 +1054,9 @@ std::unique_ptr<Filter> createBigintValuesFilter(
         return std::make_unique<NegatedBigintValuesUsingBitmask>(
             min, max, values, nullAllowed);
       }
+      if( min == max) {
```

Comment:
Also we should check it before line 1022, no?

Reply:
Hi Jimmy,
good catch, it is more efficient to check min == max earlier.
I moved it to line# 1039, to use the min max;